terraform/aks/workspace_variables/review_aks.tfvars.json

```diff
@@ -1,11 +1,30 @@
 {
   "paas_app_environment": "review",
+  "azure_resource_prefix": "s189t01-att-rv",
   "key_vault_resource_group": "s189t01-att-rv-rg",
   "key_vault_name": "s189t01-att-rv-kv",
   "key_vault_app_secret_name": "APPLY-APP-SECRETS-REVIEW",
   "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-QA",
   "cluster": "test",
+  "cluster_properties": {
+    "test": {
+      "cluster_resource_group_name": "s189t01-tsc-ts-rg",
+      "cluster_name": "s189t01-tsc-test-aks",
+      "dns_suffix": "test.teacherservices.cloud",
+      "vnet_name": "aks-vnet-37505535"
+    },
+    "platform-test": {
+      "cluster_resource_group_name": "s189t01-tsc-pt-rg",
+      "cluster_name": "s189t01-tsc-platform-test-aks",
+      "dns_suffix": "platform-test.teacherservices.cloud",
+      "vnet_name": "aks-vnet-38857354"
+    }
+  },
+  "postgres_dns": "review.internal.postgres.database.azure.com",
```

Comment:
As discussed, these can be inferred or changed to constants

Reply:
moved to terraform.tf